docs/complexity.md

```diff
@@ -51,6 +51,15 @@ In next step, you need to integrate `graphql-query-complexity` with your graphql
         onComplete: (complexity: number) => {
           console.log("Query Complexity:", complexity);
         },
+        estimators: [
+          fieldConfigEstimator(),
```

Comment:
Here should be comment that this one is mandatory to make it works

Reply:
done